src/main/java/gdx/liftoff/ui/dialogs/FullscreenDialog.java

```diff
@@ -5,6 +5,7 @@
 import com.badlogic.gdx.scenes.scene2d.ui.*;
 import com.badlogic.gdx.scenes.scene2d.ui.Window.WindowStyle;
 import com.badlogic.gdx.utils.Align;
+import com.badlogic.gdx.utils.Array;
```

Comment:
Sneaky sneaky unused import... I'll get the imports cleaned up globally at some point.